shell/browser/electron_browser_main_parts.cc

```diff
@@ -375,8 +376,24 @@ void ElectronBrowserMainParts::PostDestroyThreads() {
   fake_browser_process_->PostDestroyThreads();
 }
 
+#if defined(OS_LINUX)
+// static
+base::Optional<std::string> ElectronBrowserMainParts::GetGDKBackend() {
```

Comment:
It should return `base::Optional<std::string>&` otherwise it could not be modified.